### PR TITLE
LPS-147197 Add test AdminCanRejectRemotePortlet

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Create automated tests for Remote Apps Workflow.

**Test Plan**
Given: Workflow Approval process applied to remote apps
And Given: remote app with pending status
When: administrator rejects remote app
Then: ~rejected~ pending status is applied (according to [intended behavior](https://issues.liferay.com/browse/LPS-147219?focusedCommentId=2687428&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2687428) outlined by the Workflow team)
And Then: Cannot add remote portlet to page

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147197

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsWorkflow.testcase [hunk 1]
```diff
@@ -96,11 +96,82 @@ definition {
...
+
+			RemoteAppsWorkflow.assertTableStatus(
+				entryName = "Test Remote App",
+				entryStatus = "Pending");
+		}
+
+		task ("And Then assert remote app portlet cannot be added to page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Page");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsWorkflow.testcase:160]: Good use of JSON as part of the assertion.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsWorkflow.testcase [hunk 2]
```diff
@@ -96,11 +96,82 @@ definition {
...
+				locator1 = "Notifications#NOTIFICATIONS_WORKFLOW_TITLE",
+				value1 = "userfn userln sent you a Client Extension Entry for review in the workflow.");
+
+			LexiconEntry.gotoEllipsisMenuItem(menuItem = "Assign to Me");
+
+			Workflow.confirmWorkflowAction();
+
+			LexiconEntry.gotoEllipsisMenuItem(menuItem = "Reject");
+
+			Workflow.confirmWorkflowAction();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsWorkflow.testcase:146]: I feel like these steps are pretty common in Workflow that there's a good chance there's already a macro that exists for this. Could you double check? Go ahead and forward either after incorporating changes or no changes if not available.

**Review by john-co — APPROVED** [review]

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**